src/main/java/frc/robot/vision/cameras/limelight/Limelight.java

```diff
@@ -66,16 +72,16 @@ public Limelight(String name, String logPathPrefix, Pose3d robotRelativeCameraPo
 	}
 
 	public void updateObjectDetection() {
```

Comment:
```suggestion
	public void updateNuralDetection() {
```
same for some names in the func